db/migrate/20180109003825_add_reply_order_to_replies.rb

```diff
@@ -0,0 +1,19 @@
+class AddReplyOrderToReplies < ActiveRecord::Migration[5.1]
+  def up
+    add_column :replies, :reply_order, :integer
```

Comment:
I have a strong, strong suspicion you're going to need an index on this if you're doing ordering operations on it all over the place.